larsim/EventGenerator/PhotonGen_module.cc

```diff
@@ -154,26 +154,6 @@ namespace evgen {
     fZmax = CryoBounds[5];
     std::cout << "Cryo Boundaries:" << std::endl;
     std::cout << "Xmin: " << fXmin << " Xmax: " << fXmax << " Ymin: " << fYmin << " Ymax: " << fYmax << " Zmin: " << fZmin << " Zmax: " << fZmax << std::endl;
```

Comment:
This line is too long.  You should see this:

```console
$ which format-code
/products/cetmodules/v3_20_00/bin/format-code

$ format-code larsim/EventGenerator/PhotonGen_module.cc 
=> Applying clang-format 7.0.0 to 1 files
Changed 1 files

$ git diff
diff --git a/larsim/EventGenerator/PhotonGen_module.cc b/larsim/EventGenerator/PhotonGen_module.cc
index e0fb234b..4b1d4be3 100644
--- a/larsim/EventGenerator/PhotonGen_module.cc
+++ b/larsim/EventGenerator/PhotonGen_module.cc
@@ -153,7 +153,8 @@ namespace evgen {
     fZmin = CryoBounds[4];
     fZmax = CryoBounds[5];
     std::cout << "Cryo Boundaries:" << std::endl;
-    std::cout << "Xmin: " << fXmin << " Xmax: " << fXmax << " Ymin: " << fYmin << " Ymax: " << fYmax << " Zmin: " << fZmin << " Zmax: " << fZmax << std::endl;
+    std::cout << "Xmin: " << fXmin << " Xmax: " << fXmax << " Ymin: " << fYmin << " Ymax: " << fYmax
+              << " Zmin: " << fZmin << " Zmax: " << fZmax << std::endl;
     run.put(std::make_unique<sumdata::RunData>(geo->DetectorName()));
 
     return;
```

Reply:
Apply that change, push it to the branch, and then code-checks step should succeed.

Reply:
Looks like the version of format-code I was using is not up-to-date:
 which format-code
/cvmfs/larsoft.opensciencegrid.org/products/cetmodules/v3_16_00/bin/format-code
How can I specify the version of format-code?

Reply:
it's clear! Thanks!